javascript/first-js-oop-project.md

```diff
@@ -94,7 +94,7 @@ $.each( obj, function( key, value ) {
         // insert final html into #videos...
     $("#videos").html(html); });```
 
-- Create the html file name ***index.html***. 
+- Create the html file named ***index.html***. 
```

Comment:
the -> an
html -> HTML

Reply:
@yati1998 Just one more change...

Reply:
@Kundan28  the above line should be "Create the HTML" ??

Reply:
"Create an HTML" seems good to me..

Reply:
okay. Thanks